### PR TITLE
feat(cli): add `--devtools` option for build and generate

### PR DESCRIPTION
**Pull request by mottox2**
## Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
This PR aimed to resolve issue #4117.

This PR adds  `devtools` option to nuxt-generate.
Whee `devtools` option is enabled, devtool is enabled and debugging is easy.

## Checklist:
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly. (PR: nuxt/docs#991)
- [x] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

File: packages/cli/src/commands/generate.js [hunk 1]
```diff
@@ -12,6 +12,18 @@ export default {
       default: true,
       description: 'Only generate pages for dynamic routes. Nuxt has to be built once before using this option'
     },
+    development: {
```
Comment (pi0) [inline packages/cli/src/commands/generate.js]: To prevent confusions with `dev-development` i suggest renaming this option to `dev-tool`

**Comment by pi0** [comment]
@mottox2 This option would be useful for `nuxt build` too :D

**Comment by mottox2** [comment]
@pi0 
Thanks you for reviewing. I renamed it.

> This option would be useful for nuxt build too :D

I would like to do this change. Should I change this in this PR or another PR?

**Comment by pi0** [comment]
@mottox2 thanks for rename. Your choice if want to continue works on this PR. The refactor includes creating a file in `cli/src/options/build.js`, move added option there and import build in both `generate` and `build`. BTW no pressure if you want we can merge this as-as :)

**Review by pi0 — DISMISSED** [review]
Duplicate logic can be refactored. But safe to delegate this refactoring to later :)

**Review by clarkdo — DISMISSED** [review]
LGTM

**Comment by manniL** [comment]
Slightly improved the cmd descriptions

**Review by TheAlexLichter — APPROVED** [review]

**Review by pi0 — APPROVED** [review]

**Review by aldarund — APPROVED** [review]

**Comment by clarkdo** [comment]
@mottox2 Thanks 😸

**Pull request merged**